configs/mobilevit/metafile.yml

```diff
@@ -0,0 +1,60 @@
+Collections:
+  - Name: MobileViT
+    Metadata:
+      Training Data: ImageNet-1k
+      Architecture:
+        - MobileViT Block
+    Paper:
+      URL: https://arxiv.org/abs/2110.02178
+      Title: MobileViT Light-weight, General-purpose, and Mobile-friendly Vision Transformer
+    README: configs/mobilevit/README.md
+
+Models:
+  - Name: mobilevit_small_in1k
```

Comment:
If the weight is converted from the official repo, it should be like `mobilevit-small_3rdparty_in1k`

Reply:
done